### PR TITLE
added the slash command /create - to create a new channel

### PR DESCRIPTION
**Pull request by Deepakkothandan**
@RocketChat/core 

The PR adds a new slash command /open, which creates a new channel.

### CODE REVIEW

**Comment by rodrigok** [comment]
Why `/open` and not `/create channel`?

**Comment by Deepakkothandan** [comment]
I made it to be /open so its compatible with slack

**Comment by graywolf336** [comment]
It'd be neat if `/open #channel-that-exists` opens that channel instead of displaying the error message.

Also, may I ask why you've got it returning this `return Open;` at the end? As I don't think that is required, unless there's something I'm missing

**Comment by rodrigok** [comment]
@Deepakkothandan I don't think the `/open` as the best name if the comment don't open the channel if it already exists, your description for it is `Create a new channel` while in Slack is `Open a channel`, so the main goal of this command should be to open a existent channel and if it not exists send user to the channel creation screen.

**Comment by rodrigok** [comment]
@Deepakkothandan what do you think? I think that is what the users will expect. 😄

**Comment by Deepakkothandan** [comment]
@rodrigok Thank you for pointing it out, and I misunderstood the slack command,  I will rename the command to create and open another PR for /open, ok ?

**Comment by rodrigok** [comment]
@Deepakkothandan awesome, thanks 😸

**Comment by Deepakkothandan** [comment]
@rodrigok Rename the command to /create, which is for creating a new channel

File: packages/rocketchat-slashcommands-open/server.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+function Create(command, params, item) {
+	var channel, room, user;
+	if (command !== 'open' || !Match.test(params, String)) {
```
Comment (rodrigok) [inline packages/rocketchat-slashcommands-open/server.js]: Can you rename the command here?

**Comment by rodrigok** [comment]
@Deepakkothandan can you rename the folder too?

**Comment by rodrigok** [comment]
LGTM

**Comment by rodrigok** [comment]
@RocketChat/core I need one more review here 😸

**Comment by sampaiodiego** [comment]
LGTM

**Pull request merged**